dataherald/api/__init__.py

```diff
@@ -49,7 +49,9 @@ def add_description(
         pass
 
     @abstractmethod
-    def add_golden_records(self, golden_records: List[GoldenRecordRequest]) -> bool:
+    def add_golden_records(
+        self, namespace: str, golden_records: List[GoldenRecordRequest]
+    ) -> list[dict]:
```

Comment:
Sorry I believe I wasn't clear about what I said earlier about separating them into commits. I mean separate them in two PRs  so that you can unblock other people faster, but it seems like the namespace is closely related to the crud endpoints with golden_sql so its fine. For future its better to have them separated